crates/napi/src/next_api/utils.rs

```diff
@@ -477,9 +477,8 @@ pub fn subscribe<T: 'static + Send + Sync, F: Future<Output = Result<T>> + Send,
 
             let status = func.call(
                 result.map_err(|e| {
-                    let error = PrettyPrintError(&e).to_string();
-                    log_internal_error_and_inform(&error);
-                    napi::Error::from_reason(error)
+                    log_internal_error_and_inform(&e);
+                    napi::Error::from_reason(PrettyPrintError(&e).to_string())
```

Comment:
Why was this changed? Do we no longer want to pretty print the error in CI?

Reply:
`log_internal_error_and_inform` was changed to accept an Error, which does its own internal pretty printing the same way this does.